api/internal/git/repospec_test.go

```diff
@@ -89,6 +92,10 @@ func TestNewRepoSpecFromUrlErrors(t *testing.T) {
 			"htxxxtp://github.com/",
 			"url lacks host",
 		},
+		"bad_scp": {
+			"git@local/path:file/system",
+			"url lacks repoPath",
+		},
```

Comment:
Can we move this into `TestNewRepoSpecFromUrl_Smoke`? This test originally checked that `extractHost` failed on not being able to find a host delimiter because `:` is behind a `/`. Now that we use `/` as the delimiter instead, this test no longer makes sense. 

If we still want to document the behavior that we only detect `:`s not preceded by `/`, the `TestNewRepoSpecFromUrl_Smoke` test would look something like this:
```go
		{
			name:      "colon behind slash not scp delimiter",
			input:     "git@gitlab.com/user:name/YOUR-REPOSITORY.git",
			cloneSpec: "git@gitlab.com",
			absPath:   notCloned.String(),
			repoSpec: RepoSpec{
				Host:      "git@gitlab.com/",
				OrgRepo:   "user:name/YOUR-REPOSITORY",
				Path:      "",
				GitSuffix: ".git",
			},
		},
```

Reply:
Agreed, that's much better! Updated.